### DISCUSSION TITLE
Base Item Limit

### DISCUSSION THREAD

**Discussion by lthon09** [deta/cloud-docs#297] [Help] [answered]
I've heard that the limit of a key is **400KB** and I'm planning on creating multiple keys so that if the limit is reached, there would be a bypass. Is there a solution to this without doing that? (I have to do a voting list)

**Comment by jnsougata** [comment, marked as answer]
you can always use drive

**Reply by lthon09** [reply]
im not storing files

**Reply by jnsougata** [reply]
if your `strings` / `jsons` w/e .... if those are that large.... you should store those in a file rather than chunking and storing with serialized records inside base

**Reply by lthon09** [reply]
my data isnt that large yet, but it can. is there any other solutions that uses base?

**Reply by abdelhai** [reply]
if your data is above a couple KB, then it _is_ a file, 400kb is already A LOT for a db entry, or we're missing something...

**Reply by lthon09** [reply]
just wanna make sure for the long term, if anyone ever uses my project :p. do you have any better ideas for storing the likes of a post? (weird question ye), since im just storing and stacking up the usernames of the people who have upvoted the post here so im worried

**Reply by abdelhai** [reply]
use a votes base. each vote entry references the user (by id, username, etc) and references the post via id, etc

**Reply by lthon09** [reply]
oh shit didnt think of that, thank you
